solo-io/gloo
# test/ginkgo: ensure admin server port is not selected dynamically

# Description

Always skip 9095. when selecting a port dynamically

# Context
The issue holds the context. It caused issues for me running tests locally, where requests were sent to the Gloo admin server instead of my local proxy. As a result, the test failed locally and caused toil trying to debug, when the issue was that it was sending the request to the wrong backend.


## Testing steps
Run all tests:
```
TEST_PKG=./test/e2e INVALID_TEST_REQS=skip make test
```
in the logs, you will see:
```
Retrying because : port 9095 is in deny list
```
and the next EnvoyInstance uses 9096.

## Notes for reviewers
-

# Checklist:
